lib/widget.dart

```diff
@@ -372,6 +372,7 @@ class _StoriesList extends StatelessWidget {
                           padding: EdgeInsets.zero,
                           height: 20,
                           textStyle: TextStyle(
+                            fontSize: 16,
```

Comment:
**thought**: I was hoping this could be done externally through the theme.  Doesn't look they even use the theme though with the hardcoded TextStyles.

Reply:
**suggestion**: Maybe some padding around these would make them a little easier tap target.

Reply:
Yes I agree, I'm going to add like 5 or 7 points of padding to the top!

Reply:
Done!